### PR TITLE
SAK-30997: Checkbox alignment issues in download all/upload all screen

### PR DESCRIPTION
**Pull request by ramonchu2k**

### CODE REVIEW

**Comment by alejandrogj** [comment]
LGTM

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm [hunk 1]
```diff
@@ -43,69 +43,56 @@
...
 			</p>
 				
 			#if (!$download)
 				<h4>$tlang.getString("uploadall.releaseOption")</h4>
-				<div class="checkbox  indnt2">
+				<p class="radio">
 					## release uploaded info
-					<input type="radio" name="release" id="release_true" value="true" #if($!releaseGrades)checked="checked"#end>
-					<label for="release_true">$tlang.getString("uploadall.releaseOption.yes")</label>
+					<label for="release_true"><input type="radio" name="release" id="release_true" value="true" #if($!releaseGrades)checked="checked"#end> $tlang.getString("uploadall.releaseOption.yes")</label>
```
Comment (juanjmerono) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm]: Close the input tag here.

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm [hunk 2]
```diff
@@ -43,69 +43,56 @@
...
 					<br/>
 					## NOT release uploaded info
-					<input type="radio" name="release" id="release_false" value="false" #if(!$!releaseGrades)checked="checked"#end>
-					<label for="release_false">$tlang.getString("uploadall.releaseOption.no")</label>
-				</div>
+					<label for="release_false"><input type="radio" name="release" id="release_false" value="false" #if(!$!releaseGrades)checked="checked"#end> $tlang.getString("uploadall.releaseOption.no")</label>
```
Comment (juanjmerono) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm]: Close the input tag here.

**Comment by ramonchu2k** [comment]
IHILTY

**Comment by ramonchu2k** [comment]
I've close those inputs.

**Comment by juanjmerono** [comment]
Cool, thank you !!

**Pull request merged**